cobra/util/context.py

```diff
@@ -42,7 +42,7 @@ def __exit__(self, type, value, traceback):
 
     def execute(self, bookmark=None):
         """ Execute up to the given entry in the history.
-        If no entry is specifed, executes the last entry.
+        If no entry is specified, executes the last entry.
```

Comment:
Honestly we could remove the bookmark argument if we're not going to use it.
I included it because I wondered if we'd need it for the nested managers, but ended up using independent stacks instead.